app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<h1>Recipe details</h1>
+<h1><%= @recipe.name %></h1>
+<p>Preparation time: <%= @recipe.preparation_time %> min</p>
+<p>Cooking time: <%= @recipe.cooking_time %> min</p>
+<p>Steps: <%= @recipe.description %></p>
+<% if can? :update, @recipe %>
+<%= button_to(@recipe.public? ? "Make Private" : "Make Public", toggle_recipe_public_path(@recipe), method: :put) %>
+    <%= link_to 'Generate Shopping List', "/shopping_list" %>
+    <%= link_to 'Add Ingredients', "/recipes/#{@recipe.id}/recipe_foods/new" %>
+<% end %>
+<div>
+<table>
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <% if can? :destroy, @recipe %>
+        <th>Actions</th>
+      <% end %>
+    </tr>
+  </thead>
+  <tbody>
+    <% @foods.each do |food| %>
+      <tr>
+        <td><%= link_to food.name %></td>
+        <% recipe_food = food.recipe_foods.find_by(food_id: food.id) %>
+        <% if recipe_food %>
+          <td><%= food.quantity %></td>
+          <td>
+            $<%= food.price * recipe_food.quantity %>
+          </td>
+        <% else %>
+          <td colspan="2">No quantity available</td>
+        <% end %>
+        <% if can? :destroy, @recipe %>
+          <td>
+            <%= link_to 'Edit', edit_recipe_recipe_food_path(@recipe, recipe_food) %>
+            <%= button_to 'Remove', recipe_recipe_food_path(@recipe, recipe_food), method: :delete %>
+          </td>
+        <% end %>
```

Comment:
- [ ] It would be nice if we keep to the given wireframe
your design :point_down: 
![Screenshot from 2023-12-02 12-40-18](https://github.com/Kaiserabbas/food_recipe/assets/66097371/d4fd3821-a992-46b3-a3ed-8f89a215d1af)
given wireframe :point_down: 
![Screenshot from 2023-12-02 12-41-23](https://github.com/Kaiserabbas/food_recipe/assets/66097371/0c8c1ddc-85d9-409a-b161-c3f6b0e06239)